db/db_impl.cc

```diff
@@ -607,6 +610,26 @@ int DBImpl::FindMinimumEmptyLevelFitting(ColumnFamilyData* cfd,
   return minimum_level;
 }
 
+Status DBImpl::FlushWAL(bool sync) {
```

Comment:
I think SyncWAL can go to db_impl_write.cc too. But, yeah, let's leave them here.